modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,315 @@
+ }'
+----
+<1> Specify the same `target_namespace` where the `SriovNetwork` object is created.
+
+. Apply the `yaml` by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f tap-example.yaml
+----
+
+. Create a performance profile by following the guidance in "Creating a performance profile".
+
+. Create the following `Pod` spec, and then save the YAML in the `dpdk-pod-rootless.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dpdk-app
+  namespace: test-namespace <1>
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {"name": "sriov-network", "namespace": "test-namespace"},
+      {"name": "tap-one", "interface": "ext0", "namespace": "test-namespace"}]'
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "worker-1"
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+  - name: testpmd
+    image: <DPDK_image> <2>
+    securityContext:
+      capabilities:
+        drop: ["ALL"] <3>
+        add: <4>
+          - IPC_LOCK
+          - SYS_RESOURCE
+          - NET_RAW #for mlx only <5>
+          - NET_ADMIN
+      fsGroup: 801 <6>
+      runAsUser: 1001 <7>
+      runAsGroup: 1001 <8>
+      privileged: false <9>
+      allowPrivilegeEscalation: true <10>
+      runAsNonRoot: true <11>
+    volumeMounts:
+    - mountPath: /dev/hugepages <12>
+      name: hugepage
+    resources:
+      limits:
+        openshift.io/mlxnics: "1" <13>
+        memory: "1Gi"
+        cpu: "4" <14>
+        hugepages-1Gi: "4Gi" <15>
+      requests:
+        openshift.io/mlxnics: "1"
+        memory: "1Gi"
+        cpu: "4"
+        hugepages-1Gi: "4Gi"
+    command: ["sleep", "infinity"]
+  runtimeClassName: performance-cnf-performanceprofile <16>
+  volumes:
+  - name: hugepage
+    emptyDir:
+      medium: HugePages
+----
++
+--
+<1> Specify the same `target_namespace` where the `SriovNetwork` object `intel-dpdk-network` is created. If you want to create the pod in a different namespace, change `target_namespace` in both the `Pod` spec and the `SriovNetwork` object.
+<2> Specify the DPDK image which includes your application and the DPDK library used by application.
+<3> Dropping all capabilities (`ALL`) from the container's securityContext means that the container will not have any special privileges beyond what is necessary for normal operation.
+<4> Specify additional capabilities required by the application inside the container for hugepage allocation, system resource allocation, and network interface access the Virtual Function I/O (VFIO) device.
+<5> Mellanox network interface controller (NIC) requires the `NET_RAW` capability.
+<6> Defines the group ID (GID) that the pod's containers use to access shared volumes' file systems. This needs to be set to 801 to access the VFIO device.
++
+[NOTE]
+====
+Alternatively add the group 801 to the non-root user in the image. The user 1001(`dpdk-user`) in `registry.redhat.io/openshift4/dpdk-base-rhel8` has this group added.
```

Comment:
this should be in bullet 6. Together. So, mention that instead of using this parameter (fsGroup), they can do the following .... User is "default" not "dpdk-user."